flamedisx/lxe_blocks/energy_spectrum.py

```diff
@@ -120,6 +119,9 @@ def validate_fix_truth(self, d):
             # Assume fix_truth is a one-line dataframe with at least
             # cols columns
             return d[cols].iloc[0].to_dict()
+        elif isinstance(d, pd.Series):
+            cols = ['x', 'y', 'z', 'r', 'theta', 'event_time', 'drift_time']
```

Comment:
Duplicates line 118; you can define it outside the if

Reply:
Done